docs/specification/external-terms.md

```diff
@@ -20,3 +20,32 @@ http://xmlns.com/foaf/spec/
 https://www.w3.org/TR/vocab-org/
 
 * [`org:Organization`](https://www.w3.org/TR/vocab-org/#org:Organization)
+
+## QUDT
+
+http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html
+
+* [`qudt:QuantityValue`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Classes)
+* [`qudt:unit`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Properties)
+* [`qudt:numericValue`](http://qudt.org/doc/2016/DOC_SCHEMA-QUDT-v2.0.html#Properties)
+
+### UNIT
+
+http://qudt.org/doc/2017/DOC_VOCAB-UNITS-BASE.html
+
+* [`unit:Number`](http://qudt.org/doc/2017/DOC_VOCAB-UNITS-BASE.html#Instances)
+
+## TIME
+
+https://www.w3.org/TR/owl-time
+
+* [`time:inXSDDateTimeStamp`](https://www.w3.org/TR/owl-time/#time:inXSDDateTimeStamp)
+* [`time:hasDuration`](https://www.w3.org/TR/owl-time/#time:hasDuration)
+* [`time:before`](https://www.w3.org/TR/owl-time/#time:before)
+* [`time:after`](https://www.w3.org/TR/owl-time/#time:after)
+* [`time:intervalDuring`](https://www.w3.org/TR/owl-time/#time:intervalDuring)
+* [`time:intervalStarts'](https://www.w3.org/TR/owl-time/#time:intervalStarts)
+* [`time:intervalFinishes'](https://www.w3.org/TR/owl-time/#time:intervalFinishes)
```

Comment:
That's a lot to expect apps to support.  I think we should see what is used by known real use cases, and include them, then add as needed.  After all, anyone can use any of them if they want, and have coordinated with their communication partners.

Reply:
Any app only will support what functionality of that app requires. Here I don't consider server hosting some dataset as an app since it can't never know what various apps reading and writing from it will need or will not need and can't assume how many different properties apps will use. Still I only left those which we already encountered in our recent discussions.